src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3810,4 +3809,18 @@ public static void setMinimumKeyspaceRF(int value) throws ConfigurationException
 
         conf.minimum_keyspace_rf = value;
     }
+
+    public static boolean getUseStatementsEnabled()
+    {
+        return conf.use_statements_enabled;
+    }
+
+    public static void setUseStatementsEnabled(boolean enabled)
+    {
+        if (enabled != conf.use_statements_enabled)
+        {
+            logger.info("Setting use_statements_enabled to {}", enabled);
```

Comment:
heh... `StorageProxy` doesn't do the logging, but `StorageService` does... this patch is consistent with StorageProxy, which calls this...